src/components/DataSourceForm/index.tsx

```diff
@@ -0,0 +1,136 @@
+import { Suspense } from "react";
+
+import DataSourceSelection from "@/components/DataSourceSelection";
+import type { DataSource, DynamicForm } from "@/types/dataSource";
+
+import DataSourceSetup from "../DataSourceSetup";
```

Comment:
let's use `@/components/componentName` for all paths, it's easier to debug and change